tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -99,6 +99,13 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.DisabledIntegrationNames"/>
         public const string DisabledIntegrations = "DD_DISABLED_INTEGRATIONS";
 
+        /// <summary>
+        /// Configuration key for a list of ActivitySource names (supports globbing) that will be disabled.
+        /// Default is empty (all ActivitySources will be subscribed to by default).
+        /// Supports multiple values separated with commas.
+        /// </summary>
+        public const string DisabledActivitySources = "DD_DISABLED_ACTIVITY_SOURCES";
```

Comment:
We probably want to use the standard `DD_TRACE_` prefix here.

(Don't follow `DD_DISABLED_INTEGRATIONS` above: that's a very old setting from before we had naming conventions. 😅)

```suggestion
        public const string DisabledActivitySources = "DD_TRACE_DISABLED_ACTIVITY_SOURCES";
```

Reply:
Don't forget to add this to the configuration telemetry intake's allow-list.

Reply:
https://github.com/DataDog/dd-go/pull/148984

Thanks! Changed the name and made that PR (draft at the moment)

Sorry, I didn't accept your suggestion, got carried away locally :)